src/app/components/client/Modal.tsx

```diff
@@ -4,40 +4,103 @@
+import { Button } from "../server/Button";
+import { StaticImageData } from "next/image";
+import { metropolis } from "../../../../src/app/fonts/Metropolis/metropolis";
+import { Inter } from "next/font/google";
+import { getL10n } from "../../functions/server/l10n";
+const inter = Inter({ subsets: ["latin"], variable: "--font-inter" });
 
 export type Props = {
   isOpen: boolean;
   onClose: () => void;
-  content: ReactElement;
+  image: StaticImageData;
+  headline: string | ReactElement;
+  body: string | ReactElement;
 };
 
 export const Modal = (props: Props) => {
   if (!props.isOpen) {
-    return null; // Render nothing if the modal is closed
+    return null;
   }
 
   return (
-    <div
-      role="dialog"
-      aria-modal="true"
-      aria-label="Modal"
-      className={styles.modalOverlay}
-    >
-      <div className={styles.modal}>
-        <div className={styles.modalContent}>
-          {props.content}
-          <button
-            aria-label="Close modal"
-            className={styles.closeButton}
-            onClick={props.onClose}
-          >
-            <CloseBtn alt="" width="14" height="14" />
-          </button>
+    <OverlayContainer>
+      <DialogBox
+        isOpen={props.isOpen}
+        headline={props.headline}
+        body={props.body}
+        onClose={() => props.onClose()}
+        isDismissable={true}
+        image={props.image}
+        exitBtn={true}
+      ></DialogBox>
+    </OverlayContainer>
+  );
+};
+
+type DialogBox = {
+  headline: string | ReactElement;
+  body: string | ReactElement;
+  isOpen: boolean;
+  image: StaticImageData;
+  onClose: () => void;
+  exitBtn: boolean;
+};
+
+const DialogBox = (props: DialogBox & AriaOverlayProps) => {
+  const wrapperRef = useRef<HTMLDivElement>(null);
+  const { overlayProps, underlayProps } = useOverlay(props, wrapperRef);
+  usePreventScroll();
+  const { modalProps } = useModal();
+  const { dialogProps } = useDialog({}, wrapperRef);
+  const cancelButtonRef = useRef<HTMLButtonElement>(null);
+  const cancelButton = useButton(
+    { onPress: () => props.onClose() },
+    cancelButtonRef
+  );
+  const l10n = getL10n();
+
+  return (
+    <div className={styles.modalUnderlay} {...underlayProps}>
+      <FocusScope contain restoreFocus autoFocus>
+        <div
+          {...overlayProps}
+          {...dialogProps}
+          {...modalProps}
+          ref={wrapperRef}
+          className={`${styles.modal} ${inter.variable} ${metropolis.variable}`}
+        >
+          {props.exitBtn ? (
+            <button
+              className={styles.dismissButton}
+              {...cancelButton.buttonProps}
+              ref={cancelButtonRef}
+            >
+              <CloseBtn alt="" width="14" height="14" />
+            </button>
+          ) : null}
+          <img src={props.image.src} alt="" />
+          <dl className={styles.modalContent}>
+            <dt>{props.headline}</dt>
+            <dd>{props.body}</dd>
+          </dl>
```

Comment:
This isn't really a list since there's just one element, so I think a regular `<h2>` and `<p>` would do as well.

```suggestion
          <div className={styles.modalContent}>
            <h2>{props.headline}</h2>
            <p>{props.body}</p>
          </div>
```